src/state/nftMarket/types.ts

```diff
@@ -115,6 +115,13 @@ export interface Collection {
   owner: string
   totalSupply: BigNumberish
   verified: boolean
+  avatar: string
+  banner: {
+    large: string
+    small: string
+  }
+  createdAt: string
```

Comment:
line 123 and 124 need to be removed to avoid confusion with subgraph data

Reply:
Updated.